http.go

```diff
@@ -41,6 +44,7 @@ func ListenAndServe(addr string) error {
 	r.HandleFunc("/api/mailbox/{box}", mailboxHandler).Methods("GET")
 	r.HandleFunc("/api/mailbox/{box}/{mid}", messageHandler).Methods("GET")
 	r.HandleFunc("/api/mailbox/{box}/{mid}/{attachment}", attachmentHandler).Methods("GET")
+	r.HandleFunc("/api/mailbox/{box}/{mid}/{attachment}/view", attachmentViewHandler).Methods("GET")
```

Comment:
Is this only related to _viewing_ a sent Winlink form not filling a winlink form based on the forms users can pull in from https://www.winlink.org/content/how_manually_update_standard_templates_version_101200 ? 
I had some ideas on sending as it looks like that is somewhat web based now, don't want to reinvent.

Reply:
Hi, this code is just for viewing forms.

Reply:
That's what I thought. As soon as I can [get this to build locally](https://github.com/la5nta/pat/issues/195) I have some ideas on adding filling out the forms and sending them, feature. If you haven't added that already?